scripts/data/alloprof/alloprof.py

```diff
@@ -0,0 +1,125 @@
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
```

Comment:
I guess this file is already here so maybe no need to merge it into mteb https://huggingface.co/datasets/lyon-nlp/alloprof/blob/main/alloprof.py

Reply:
Indeed. The builder scripts, along with the processing scripts used to initially build the HF dataset have been moved to their respective HF dataset repos. Same thing for data/syntec

Reply:
Done.